### PR TITLE
Added initial configuration for poetry and the venv. Also added the .…

### PR DESCRIPTION
**Pull request by Eric-Butcher**
…gitgnore and updated the README with instructions to use poetry.

### CODE REVIEW

**Review by sumanyuj — DISMISSED** [review]
LGTM!

**Comment by sumanyuj** [comment]
The main thing here is to have the Django core dependency, which you have here.

**Review by giagermain — DISMISSED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "project-evergreen"
+version = "0.1.0"
+description = "Project for CSE312 in Fall 2024."
+authors = ["Eric-Butcher <ebutcher66@gmail.com>"]
```
Comment (WeirdAlex03) [inline pyproject.toml]: I think this is missing a few entries. Otherwise lgtm

**Review by WeirdAlex03 — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (sumanyuj) [reply pyproject.toml:5]: will revise. I admit I just copied these files from Eric.

Comment (Eric-Butcher) [reply pyproject.toml]: I added everyone as authors using school email addresses.

**Review by WeirdAlex03 — APPROVED** [review]

**Pull request merged**